src/lib389/lib389/configurations/config_002003000.py

```diff
@@ -0,0 +1,134 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2020 Red Hat, Inc.
```

Comment:
It was not the best year, lets forget it :D

Reply:
Also fixed :)